Snowflake.Data/Core/Session/SFSessionParameter.cs

```diff
@@ -14,5 +14,6 @@ internal enum SFSessionParameter
         QUERY_CONTEXT_CACHE_SIZE,
         DATE_OUTPUT_FORMAT,
         TIME_OUTPUT_FORMAT,
+        CLIENT_STORE_TEMPORARY_CREDENTIAL,

```

Comment:
It's added because the server only includes a token in the response if `CLIENT_STORE_TEMPORARY_CREDENTIAL` is included in the session parameters and set to true

It's possible to unify the names but it seems the server expects the parameter name to be `CLIENT_STORE_TEMPORARY_CREDENTIAL` or else it doesn't include a token. Other connectors use the same name for the parameter as well:
- [ODBC](https://github.com/snowflakedb/snowflake-odbc/blob/ea594fcb4f74711b35972221124fed5f36d44e6e/Source/Snowflake.h#L182)
- [JDBC](https://github.com/snowflakedb/snowflake-jdbc/blob/e95bba2816769fc59bfb3f2e3424b449c4b9a323/src/main/java/net/snowflake/client/core/SessionUtil.java#L81)